### PR TITLE
[RF] added missing snapshot creation

### PR DESCRIPTION
**Pull request by cburgard**
# This Pull request:

Fixes a bug where snapshots were not imported from JSON.

## Checklist:

- [x] tested changes locally
- [ ] updated the docs (if necessary)

### CODE REVIEW

**Comment by cburgard** [comment]
@guitargeek please take a look at this small bugfix. does this qualify to be backported?

**Comment by cburgard** [comment]
@guitargeek please take a look at this small bugfix. does this qualify to be backported?

File: roofit/hs3/src/RooJSONFactoryWSTool.cxx [hunk 1]
```diff
@@ -1728,13 +1728,16 @@ void RooJSONFactoryWSTool::importAllNodes(const RooFit::Experimental::JSONNode &
          std::string name = RooJSONFactoryWSTool::name(snsh);
          if (name == "fromJSON")
             continue;
+	 RooArgSet vars;
```
Comment (Axel-Naumann) [inline roofit/hs3/src/RooJSONFactoryWSTool.cxx:1731]: No tabs, please.

**Review by Axel-Naumann — CHANGES_REQUESTED** [review]

**Comment by cburgard** [comment]
done, I hope

File: roofit/hs3/src/RooJSONFactoryWSTool.cxx (hunk 1 above)
Comment (cburgard) [reply roofit/hs3/src/RooJSONFactoryWSTool.cxx]: done (I hope)

**Comment by cburgard** [comment]
Is there still anything left to be done? I see the check failing, but I can't see why.

**Comment by guitargeek** [comment]
The failing checks are unrelated, I'll merge now. Thanks for the PR :+1:

**Review by guitargeek — APPROVED** [review]

**Pull request merged**